compiler_opt/es/regalloc_trace/regalloc_trace_worker.py

```diff
@@ -60,8 +60,8 @@ def _setup_base_policy(self):
     saver.save(self._tf_base_temp_dir)
     self._tf_base_policy_path = os.path.join(self._tf_base_temp_dir, "policy")
 
-  def _copy_corpus(self, corpus_path: str,
-                   copy_corpus_locally_path: str | None) -> None:
+  def _copy_corpus(self, corpus_path: str, copy_corpus_locally_path: str | None,
+                   aux_file_replacement_flags: dict[str, str]) -> None:
```

Comment:
inlining for speed + ctxprof will need this aux_file_replacement_flags thing. Worth tagging an issue.

Reply:
Ah, true. Will add a TODO.